server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraConfTest.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.james;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+import java.nio.charset.Charset;
+
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.james.modules.mailbox.CassandraSessionConfiguration;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.google.common.base.Joiner;
+
+public class CassandraConfTest {
```

Comment:
Here is the order we currently use : 

 1/ Internal classes
 2/ static variable
 3/ static methods
 4/ final variables
 5/ (that's dirty but not always avoidable) non final variables
 6/ constructor
 7/ methods
 8/ equals, hashCode and toString

And for all of this, we order that with public, then protected, then private.